doc/en/index.html

```diff
@@ -105,33 +105,33 @@ <h3>Release Versions</h3>
           </tr>
```

Comment:
this might be for another PR, but I was wondering if the Chinese docs should be mentioned on this page? (as of `latest`) eg. https://docs.geoserver.org/latest/zhCN/user/

Reply:
Probably best as its own issue/seperate PR, yeah. I think the "best" approach would be a drop-down in the doc header allowing you to switch languages directly from a version/page, but I doubt that's feasible.

Also the Chinese docs are only really present on main right now, and only (parts) of the user guide are translated, so we'd need to determine a way of linking only the docs that do exist without looking too inconsistent. Maybe after 2.20 is released and there's actually a released version with translated docs we can add a link there.

Back when we had (partial) Spanish and French docs, I don't think they were linked anywhere either, so we don't have any defined way of representing this.